elements/lisk-cryptography/tsconfig.json

```diff
@@ -1 +0,0 @@
-../../templates/tsconfig.json.tmpl
```

Comment:
Why not consistently change in template?

Reply:
We are using some package related Type definitions here.